comp150-agile/essay.tex

```diff
@@ -0,0 +1,65 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{Do Daily Scrums Meetings Improve Interpersonal Relations Within a Game Development Team?}
+
+% Please do not change the subtitle
+\subtitle{COMP150 - Agile Development Practice}
+
+% Please put your student number in the author field
+\author{1607804}
+
+\begin{document}
+
+\maketitle
+
+
+\section{Introduction}
+There is limited research on the topic of Daily scrum meetings and whether they improve the interpersonal relations of the team. However there is a lot of theory around team effectiveness across many work disciplines which can be extrapolated to fit the principles of Agile Software Development \cite{Agile}. 
+
+In the following essay the author will outline '‘Daily Scrum meetings' this will then be followed by the main discussion which will be based upon any relevant research and the authors own conclusions. 
+
+The author, by drawing parallels between research around team working and existing literature of '‘Daily scrum meetings', hopes to demonstrate how scrum meetings can be used to achieve improved interpersonal relations. 
+
+The Author will compare research around possible positive effects of daily scrum meetings on the team and balance this with negative outcomes. 
+
+
+\section{Discussion}
+In the software development industry, daily scrums meetings have been practiced for many years. They commonly take the following format:
+
+They occur once per day during a sprint (usually a four-week process). 
+
+They involve the team gathering for a stand up meeting within a time constraint of 15 minutes. 
+
+During the meetings each team member stands and informs the rest of the team what they have done that day, what they will do tomorrow and any barriers they foresee.
+
+The aim of this being not only the resolutions of the barriers by the scrum master, but to keep the team informed on the general progress of the other members and the project as a whole. 
+
+According to various online guides  \cite{ DailyScrum, EffectiveScrum} the role of the problem solver is left to the scrum master. In contrast to this, professionals in other fields describe an effective meeting to draw "upon each members' knowledge, skills, and perspectives to solve problems and to support one another" \cite{LeadershipLesson}. Research in relevant fields found that "Relationships in virtual teams are developed and strengthened through a proactive effort to solve problems" \cite[p. 559]{tavvcar2005skills}. 

```

Comment:
I think it is important to draw a distinction between what these authors are referring to when they are discussing meetings, because they probably aren't stand-ups. Acknowledge this explicitly, and draw more attention to the comparison that you are making.